src/main/java/org/broadinstitute/hellbender/utils/SequenceDictionaryUtils.java

```diff
@@ -546,4 +494,24 @@ public static String getDictionaryAsString( final SAMSequenceDictionary dict ) {
 
         return s.toString();
     }
+
+    /**
+     * get the sequence dictionary contig list that is in the index or null if there is not index or not contigs
+     * Note: the dictionary returned will not have the contig lengths filled in {@link SAMSequenceRecord#UNKNOWN_SEQUENCE_LENGTH} is used.
+     * @return a SAMSequenceDictionary or null if the index cannot be loaded or there are no contigs in the index
+     */
+    public static SAMSequenceDictionary createSequenceDictionaryFromFeatureIndex(final File featureFile) {
+        final Index index = IndexUtils.loadTribbleIndex(featureFile);
+        if (index == null){
+            return null;
+        }
+        final List<String> seqNames = index.getSequenceNames();
+        if (seqNames == null) {
```

Comment:
What if `seqNames` is empty? Then you'll end up returning an empty dictionary, whereas the method contract states that null will be returned.